fixieai/client/agent.py

```diff
@@ -100,80 +108,120 @@ def owner(self) -> Optional[str]:
         if self._metadata is None:
             return None
         owner_username = self._metadata["owner"]["username"]
-        assert isinstance(owner_username, str)
+        assert owner_username is None or isinstance(owner_username, str)
         return owner_username
 
     @property
     def query_url(self) -> Optional[str]:
         """Return the query URL for this Agent."""
         if self._metadata is None:
             return None
-        url = self._metadata["owner"]["query_url"]
-        assert isinstance(url, str)
+        url = self._metadata["queryUrl"]
+        assert url is None or isinstance(url, str)
         return url
 
     @property
     def func_url(self) -> Optional[str]:
         """Return the func URL for this Agent."""
         if self._metadata is None:
             return None
-        url = self._metadata["owner"]["func_url"]
-        assert isinstance(url, str)
+        url = self._metadata["funcUrl"]
+        assert url is None or isinstance(url, str)
         return url
 
     @property
     def created(self) -> Optional[datetime.datetime]:
         """Return the creation timestamp for this Agent."""
         if self._metadata is None:
             return None
-        ts = self._metadata["owner"]["created"]
-        assert isinstance(ts, datetime.datetime)
-        return ts
+        ts = self._metadata["created"]
+        if ts is not None:
+            return datetime.datetime.fromisoformat(ts)
+        else:
+            return None
 
     @property
     def modified(self) -> Optional[datetime.datetime]:
         """Return the modification timestamp for this Agent."""
         if self._metadata is None:
             return None
-        ts = self._metadata["owner"]["modified"]
-        assert isinstance(ts, datetime.datetime)
-        return ts
+        ts = self._metadata["modified"]
+        if ts is not None:
+            return datetime.datetime.fromisoformat(ts)
+        else:
+            return None
 
     def get_metadata(self) -> Dict[str, Any]:
         """Return metadata about this Agent."""
 
-        query = gql(
+        if self._owner is None:
+            # Query by handle.
```

Comment:
I assume this scopes to your own user id, rather than returning everything with that handle?

Reply:
yes